lib/processCss.js

```diff
@@ -83,6 +83,11 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
 			exports[exportName] = replaceImportsInString(exports[exportName]);
 		});
 
+		function startByAlias(url) {
```

Comment:
`startsWithAlias` => `isAlias[ed]`

Reply:
I pushed a change to rename it to `isAlias`